libs/core/langchain_core/utils/_merge.py

```diff
@@ -34,6 +34,9 @@ def merge_dicts(left: Dict[str, Any], right: Dict[str, Any]) -> Dict[str, Any]:
             merged[k] = merge_dicts(merged[k], v)
         elif isinstance(merged[k], list):
             merged[k] = merged[k] + v
+        # when using Non-OpenAI chatmodels, the output and input tokens may be different between left and right
+        elif isinstance(merged[k], int):
+            merged[k] = max(merged[k], v)
```

Comment:
I have the same issue with a non-OpenAI model, except in my case it's a float instead of int.
(I am unsure whether the max operation will make sense, but if I understand correctly that this is merging streamed chat chunk responses, then it might be correct?)
Anyway if this fix makes sense, please consider also adding float for this condition.